nntrainer/layers/multi_head_attention_layer.cpp

```diff
@@ -73,10 +73,317 @@ enum AttentionParams {
   transposed_attention_output,
 };
 
-void MultiHeadAttentionLayer::finalizeCommon(InitLayerContext &context) {}
+void MultiHeadAttentionLayer::finalizeCommon(InitLayerContext &context) {
+  if (context.getNumInputs() < 2 || context.getNumInputs() > 4) {
+    throw std::runtime_error("Multi head Attention layer needs 2-4 inputs.");
```

Comment:
How about decribe more details.. what the inputs are.

Reply:
Done